src/driver/sqljs/SqljsDriver.ts

```diff
@@ -101,8 +102,17 @@ export class SqljsDriver extends AbstractSqliteDriver {
             } 
             else {
                 // browser
-                // fileNameOrLocalStorageOrData should be a local storage key
-                const localStorageContent = PlatformTools.getGlobalVariable().localStorage.getItem(fileNameOrLocalStorageOrData);
+                // fileNameOrLocalStorageOrData should be a local storage / indexedDB key
+                let localStorageContent = null;
+                if (this.options.useLocalforage) {
+                    if (window.localforage) {
+                        localStorageContent = await window.localforage.getItem(fileNameOrLocalStorageOrData);
+                    } else {
+                        throw new Error(`localforage is not defined - please import localforage.js into your site`);
```

Comment:
I guess it can be not only site

Reply:
localforage is developed for the browser, it uses indexedDB (and WebSQL as fallback), which are only available in the browser

i've added this variant, because the usage of localforage is a lot easier for (web-)app developer than importing it as node module, because there is no special webpack or browserify configuration needed